### ISSUE TITLE
Ngrok - NPM Package Unexpected token function Error

### ISSUE THREAD

**Issue by raymacz** [bubenshchykov/ngrok#118]
I used to run Ngrok package with no issues & I can run browser-sync package now w/ no problem in my localhost.
Now this is my error with Ngrok package :

         ...\gulp-dev\node_modules\ngrok\index.js:8
         async function connect (opts) {
           ^^^^^^^^

          SyntaxError: Unexpected token function
          at createScript (vm.js:56:10)
          at Object.runInThisContext (vm.js:97:10)
          at Module._compile (module.js:542:28)
          at Object.Module._extensions..js (module.js:579:10)
          at Module.load (module.js:487:32)
         at tryModuleLoad (module.js:446:12)
         at Function.Module._load (module.js:438:3)
         at Module.require (module.js:497:17)
         at require (internal/module.js:20:19)
         at Object.<anonymous> (F:\Raymacz\xampp\htdocs\wp\site1\wp-content\themes\gulp- 
         dev\gulpfile.js:17:10)

Using windows 10 w/ git bash. I am totally stuck here.   Any help would be highly appreciated. Thanks.

**Comment by bubenshchykov** [comment]
which node version you use?
also, tell me the output please
```javascript
const os = require('os');
console.log(os.platform() + os.arch());
```

**Comment by raymacz** [comment]
sorry for the late response, anyhow, my version: 

       $ node --version
        v6.12.2
       $ npm --version
       3.10.10
       $ gulp --version
       [21:06:15] CLI version 2.0.0
       [21:06:15] Local version 3.9.1

I have added your code & this is the latest output after running my gulpfile.js:

     $ gulp
     win32x64
     F:\Raymacz\xampp\htdocs\wp\site1\wp-content\themes\gulp-dev\node_modules\ngrok\index.js:8
    async function connect (opts) {
          ^^^^^^^^

    SyntaxError: Unexpected token function
       at createScript (vm.js:56:10)
       at Object.runInThisContext (vm.js:97:10)
       at Module._compile (module.js:542:28)
       at Object.Module._extensions..js (module.js:579:10)
       at Module.load (module.js:487:32)
       at tryModuleLoad (module.js:446:12)
       at Function.Module._load (module.js:438:3)
       at Module.require (module.js:497:17)
      at require (internal/module.js:20:19)
      at Object.<anonymous> (F:\Raymacz\xampp\htdocs\wp\site1\wp-content\themes\gulp-dev\gulpfile.js:31:10)

Basically, I didn't have issues before with my existing node versions & Ngrok.  After updating my node_modules, just few weeks ago, the error came out.   If I remove the ngrok package & ngrok script from my gulpfile.js using only browser-sync, it runs smoothly with no issues. 
Note: my ngrok node_modules which wasn't updated... still works (located in a different directory),.

**Comment by bubenshchykov** [comment]
Got it! As stated in readme, 

> This module uses node>=8.3.0 with async-await. For callback-based version use 2.3.0

Switch to v2.3.0 in your package.json if you can afford updating your node, and it should be good

**Comment by raymacz** [comment]
My Node is v6.12.2. I'm sorry, could you be a bit more specific what to update? Which one to switch to v2.3.0?

**Comment by bubenshchykov** [comment]
sure, few options
a) you upgrade your node to 8.3.0 or later, re-install ngrok, and it just works
b) if you can't upgrade your node, just use ngrok@2.3.0 from npm
```
npm i ngrok@2.3.0 -g
```
or if you have ngrok in package.json of a parent project, just change it to: 
```
{
    "dependencies": {
            "ngrok": "2.3.0"
     } 
}
```

I hope it helps

**Comment by raymacz** [comment]
It really helps, thank you so much. You are a savior!
